docs/source/models/gbd2017/causes/diarrheal_diseases.rst

```diff
@@ -7,27 +7,47 @@ Diarrheal Diseases
 Disease Description
 -------------------
 
+We follow GBD 2017 and "defined diarrhoeal disease episodes as three
+or more loose stools in a 24-hour period." (p. 88 of
+[GBD-2017-YLD-Capstone-Appendix-1]_).
+
 .. todo::
 
-   Adapt material from GBD capstone and other sources, e.g. [WHO]_,
+   Adapt additional material from GBD capstone and other sources, e.g. [WHO]_,
    [CDC]_, [Wikipedia]_, [GBD-2017-YLD-Capstone-Appendix-1]_
 
-The relevant ICD codes are...
-
 Modeling Diarrheal Diseases in GBD 2017
 ---------------------------------------
 
+According to the [GBD-2017-YLD-Capstone-Appendix-1]_, "There are no
+major modelling updates from GBD 2016," (p. 93) and "self-reported
+prevalence is the reference category" (p. 88).
```

Comment:
I think that this could use some more context for people who are not familiar, but it looks like more detail is coming, so looks good to me!

Reply:
Agreed